internal/controller/scheduler/scheduler.go

```diff
@@ -354,7 +354,9 @@ func (w *worker) Build(podSpec *corev1.PodSpec, skipCheckout bool, inputs buildI
 		c.Command = []string{"/workspace/tini-static"}
 		c.Args = []string{"--", "/workspace/buildkite-agent", "bootstrap"}
 
-		c.ImagePullPolicy = corev1.PullAlways
+		// The image *should* be present since we just pulled it with an init
+		// container, but weirder things have happened.
+		c.ImagePullPolicy = corev1.PullIfNotPresent
```

Comment:
Does this come with the normal caveat that if you update an image tag the new version won't be pulled? 

So you kind of need to be careful of `latest` tags or the like.

Reply:
Each distinct image is also pulled by an init container with the policy `PullAlways`, so the node should have e.g. "latest" as of a few seconds ago. 

I think changing this is an improvement with respect to concurrently-changing image tags, because if multiple command containers all use the same image, they will now use the one pulled by the init container, instead of all pulling separately and potentially having different images to one another.

Reply:
@DrJosh9000 should we include a note in the release notes to avoid surprising people when this behaviour changes?

Reply:
Will do.